Create Glossary stubs S (#3897)
ppy/osu-wiki: wiki/Storyboarding/Storyboarder/en.md

---
stub: true
tags:
  - SBer
---

> <!-- not sure if we need this -->

Author: don't think there much we can add since the term is pretty self explanatory

then again i'm not really a storyboarder myself or know much about sbs in general tbh

# Storyboarder

> A storyboarder refers to the user who makes storyboards for their own or someone else's [beatmap](/wiki/Beatmaps). They may have used the editor's design section to [storyboard](/wiki/Storyboards), used a third party application, or have done [storyboard scripting](/wiki/storyboard_scripting) (or SBS).

Reviewer: ```suggestion
A **storyboarder** is a term used to describe a user who makes storyboards for their own, or someone else's, [beatmap](/wiki/Beatmaps). Storyboarder's use the editor's design section to [storyboard](/wiki/Storyboards), used a third party application, or have done [storyboard scripting](/wiki/storyboard_scripting) (or SBS).
```

> A **storyboarder** is a term used to describe a user who makes storyboards for their own, or someone else's, [beatmap](/wiki/Beatmaps). Storyboarder's use the editor's design section to [storyboard](/wiki/Storyboards), used a third party application, or have done [storyboard scripting](/wiki/storyboard_scripting) (or SBS).

Reviewer 2: ```suggestion
A **storyboarder** is a term used to describe a user who makes storyboards for their own, or someone else's [beatmap](/wiki/Beatmaps). Storyboarders use the beatmap editor's Design section, a third party application, or [storyboard scripting](/wiki/storyboard_scripting) to create [storyboards](/wiki/Storyboards).
```
